### PR TITLE
refactor: wallet active

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-wallet/tests/store/wallet_store.js [hunk 1]
```diff
@@ -103,13 +103,52 @@ describe('wallet store', () => {
...
   it('delete wallet', () => {
-    const walletId = walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet1)
     walletStore.saveWallet(wallet2)
-    walletStore.deleteWallet(walletId)
+    walletStore.deleteWallet(wallet1.id)
+    assert.equal(walletStore.getIDList().length, 1)
     try {
-      walletStore.getWallet(walletId)
+      assert.deepEqual(walletStore.getWallet(wallet1.id), wallet1)
```
Comment (Keith-CY) [inline packages/neuron-wallet/tests/store/wallet_store.js]: In this case, the wallet1 has been removed, what's the purpose of such comparison

File: packages/neuron-wallet/tests/store/wallet_store.js [hunk 2]
```diff
@@ -103,13 +103,52 @@ describe('wallet store', () => {
...
+
+  it('delete active wallet', () => {
+    walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet2)
+    walletStore.deleteWallet(wallet1.id)
+    const activeWallet = walletStore.getActiveWallet()
+    assert.deepEqual(activeWallet, wallet2)
+  })
+
+  it('delte no active wallet', () => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/tests/store/wallet_store.js]: `delete` not `delte`, `inactive` not `no active`

File: packages/neuron-wallet/tests/store/wallet_store.js (hunk 1 above)
Comment (BaojunCZ) [reply packages/neuron-wallet/tests/store/wallet_store.js]: When `delete` is invalid and don't throw error.It should be notDeepEqual.I fixed

File: packages/neuron-wallet/tests/store/wallet_store.js (hunk 2 above)
Comment (BaojunCZ) [reply packages/neuron-wallet/tests/store/wallet_store.js]: ok

**Comment by Keith-CY** [comment]
conflicts

**Pull request merged**